src/stratis_cli/_errors.py

```diff
@@ -33,6 +33,31 @@ class StratisCliRuntimeError(StratisCliError):
     """
 
 
+# This indicates a bug and so coverage is not necessary as this error
+# is not handled.
+class StratisCliPropertyNotFoundError(StratisCliRuntimeError):  # pragma: no cover
+    """
+    Exception raised when a requested property from FetchProperties DBus interface
+    does not exist.
+    """
+
+    def __init__(self, type_name, prop_name):
```

Comment:
I'ld prefer the type name to be the name of the _D-Bus interface_ to which the object belongs. That is sort of a user unfriendly thing to report, but because we expect this to be a bug if it occurs, and since that is the precise identifier for the thing that we have failed to get the property for if we raise the exception, that seems the best choice. Any other choice would be kind of vague.

Reply:
Oops! Sorry, you already requested this. Doing this now.

Reply:
I missed this in my last round of revisions.

Reply:
Resolved.